example/README.md

```diff
@@ -109,3 +109,26 @@ For example:
 ```sh
 CGO_ENABLED=1 GOEXPERIMENT=boringcrypto go run .
 ```
+
+### FIPS mode - Windows
+
+> [!NOTE]
+> Microsoft maitains [a fork of Go](https://github.com/microsoft/go)
+> that can be used to build applications
+> which are likely to be FIPS 140-2 compliant.
```

Comment:
Why is there uncertainty here? Is it not clearly stated to be FIPS 140-2 compliant?